### PR TITLE
Add cover statuses

### PR DESCRIPTION
**Pull request by mattexdee**
This will allow features which provide cover regardless of position to include an Active Effect that applies the relevant status. It can later be checked in AC calculations and dex saves.

### CODE REVIEW

**Comment by arbron** [comment]
Screenshot 2024-09-11 at 14 10 44

**Comment by krbz999** [comment]
Wonder if it would not be cleaner to add a single condition with 3 levels. It should not be too difficult to change the name of the effect at will, and this would also prevent users from applying several different types of cover at the same time.

File: lang/en.json [hunk 1]
```diff
@@ -2891,6 +2891,8 @@
   "StatusEthereal": "Ethereal",
   "StatusExceedingCarryingCapacity": "Exceeding Carrying Capacity",
   "StatusFlying": "Flying",
+  "StatusFullCover": "Full Cover",
```
Comment (Fyorl) [inline lang/en.json:2894]: We should use the game term, which is 'Total Cover'.

**Review by Fyorl — CHANGES_REQUESTED** [review]

**Comment by arbron** [comment]
> Wonder if it would not be cleaner to add a single condition with 3 levels. It should not be too difficult to change the name of the effect at will, and this would also prevent users from applying several different types of cover at the same time.

I think three statuses is good, because unlike exhaustion you don't cycle through levels of cover, you just apply the one you want directly. What I think I'll add is that when one level of cover is added it removes any current levels of cover.

**Review by Fyorl — APPROVED** [review]

**Pull request merged**